ui-lib/src/styles/palettes/dusk.json

```diff
@@ -38,7 +38,8 @@
       "contrast": "hsla(0, 0%, 0%, 1)"
     },
     "grey": {
-      "border": "hsla(0, 1%, 23%, 1)"
+      "border": " hsla(0, 1%, 23%, 1)",
+      "borderDark": "hsla(0, 0%, 90%, 1)"
```

Comment:
Is the Dusk theme not outdated? I thought I heard that

Reply:
I have no idea how to maintain the palettes, if you are more up to date with what @LFBarreto had in mind, let me know

Reply:
Added note from my side to ask Luiz about that